x/evm/migrations/v4/migrate.go

```diff
@@ -0,0 +1,51 @@
+package v4
+
+import (
+	"github.com/cosmos/cosmos-sdk/codec"
+	storetypes "github.com/cosmos/cosmos-sdk/store/types"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+
+	v4types "github.com/evmos/ethermint/x/evm/migrations/v4/types"
+	"github.com/evmos/evmos/v11/x/evm/types"
```

Comment:
I think we only want to change the imports once we are fully migrated?
```suggestion
	"github.com/evmos/ethermint/x/evm/types"
```